tests/integration/test_fim/test_synchronization/test_sync_time.py

```diff
@@ -0,0 +1,162 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Check when the 'wazuh-syscheckd' daemon is performing a synchronization, a normal synchronization will end
+before the configured `interval` and `max_interval`.
+
+components:
+    - fim
+
+suite: synchronization
+
+targets:
+    - agent
+    - manager
+
+daemons:
+    - wazuh-syscheckd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Windows Server 2019
```

Comment:
`os_version` section contains Windows Server but in `platform` section, you only add Linux. Could you update it, please?